### ISSUE TITLE
[HOLD for payment 2024-06-21]  [$500]  mweb - Chat - App crashes when selecting an emoji

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#39234]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.58-0
**Reproducible in staging?:** y
**Reproducible in production?:** n
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Go to staging.new.expensify.com
2. Go to any chat.
3. Open emoji picker.
4. Select any emoji.

## Expected Result:
App does not crash.

## Actual Result:
App crashes.

## Workaround:
n/a

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

https://github.com/Expensify/App/assets/93399543/76656da7-14ad-4f14-8b15-ef851d41fe1d

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01da38a02293ce221e</li>
        <li>Upwork Job ID: 1773831643070943232</li>
        <li>Last Price Increase: 2024-03-29</li>
    </ul>
</details>

**Comment by kbecciv** [comment]
We think that this bug might be related to #vip-vsb

**Comment by arosiclair** [comment]
Reproduced on staging **1.4.58-0**. This only happens in with the mobile (`isSmallScreenWidth`) UI.

**Comment by arosiclair** [comment]
This seems to be coming from our live markdown code so likely related to https://github.com/Expensify/App/pull/38152. @Skalakid can you take a look at this?

```
Uncaught DOMException: Failed to execute 'getRangeAt' on 'Selection': 0 is not a valid index.
    at scrollCursorIntoView (webpack-internal:///./node_modules/@expensify/react-native-live-markdown/lib/module/web/cursorUtils.js:136:31)
    at Module.setCursorPosition (webpack-internal:///./node_modules/@expensify/react-native-live-markdown/lib/module/web/cursorUtils.js:94:3)
    at eval (webpack-internal:///./node_modules/@expensify/react-native-live-markdown/lib/module/MarkdownTextInput.web.js:414:51)
    at commitHookEffectListMount (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:23044:26)
    at commitPassiveMountOnFiber (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24811:13)
    at commitPassiveMountEffects_complete (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24776:9)
    at commitPassiveMountEffects_begin (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24763:7)
    at commitPassiveMountEffects (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:24751:3)
    at flushPassiveEffectsImpl (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:26985:3)
    at flushPassiveEffects (webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:26930:14)
```

**Comment by thienlnam** [comment]
Currently being fixed by @Skalakid

**Comment by arosiclair** [comment]
This was just a regression on staging so closing this out without payment.

**Comment by situchan** [comment]
I wasn't involved in original [PR](https://github.com/Expensify/App/pull/38152) which caused this regression.
So I am still eligible for payment.
cc: @thienlnam

**Comment by situchan** [comment]
@arosiclair can you please reopen and add `Bug` label for payment?

**Comment by arosiclair** [comment]
Makes sense. Adding the `Bug` label

**Comment by arosiclair** [comment]
Hey @abekkala can you handle payment for this? Thanks

**Comment by abekkala** [comment]
@arosiclair is @situchan the only contributor that requires payment? I'm assuming that it's for a PR Review, still at $500? or $250?

**Comment by arosiclair** [comment]
Yeah only @situchan for his C+ review. I'm not sure why Melvin set the bounty to $500 tbh. I'm pretty sure it should be $250, but it might be worth asking in #bug-zero.

**Comment by abekkala** [comment]
Ok - I'm sure that it's only $250 - Melvin is just wrong here LOL

**Comment by abekkala** [comment]
@situchan [OFFER](https://www.upwork.com/nx/wm/offer/102826826) has been sent via Upwork.

**Comment by situchan** [comment]
> TL;DR: effective today, Thursday, April 4th, 2024 the initial job price for bugs/feature requests will start at $250.
> Any issues created before this date will maintain their original stated price.

The base bounty was changed on Apr 4. The PR was reviewed on Mar 29.

https://expensify.slack.com/archives/C01GTK53T8Q/p1712256409152489

**Comment by abekkala** [comment]
@situchan payment sent and contract ended - thank you! 🎉
